pages/ref/core-cells.mdx

```diff
@@ -123,9 +212,9 @@ let fizz: Builder = b.storeSlice(s);
 extends fun storeCoins(self: Builder, value: Int): Builder;
 ```
 
-Extension function for the [`Builder{:tact}`][p].
+Extension function for the [`Builder{:tact}`][builder].
 
-Stores (serializes) an unsigned [`Int{:tact}`][int] `value` in the range $0 .. 2^{120} − 1$ into the copy of the [`Builder{:tact}`][p]. The serialization of `value` consists of a $4$-bit unsigned big-endian integer $l$, which is the smallest integer $l ≥ 0$, such that `value` $< 2^{8 * l}$, followed by an $8 * l$-bit unsigned big-endian representation of `value`. If `value` does not belong to the supported range, a range check exception is thrown. Returns that copy of the [`Builder{:tact}`][p].
+Stores (serializes) an unsigned [`Int{:tact}`][int] `value` in the range $0 .. 2^{120} − 1$ into the copy of the [`Builder{:tact}`][builder]. The serialization of `value` consists of a $4$-bit unsigned big-endian integer $l$, which is the smallest integer $l ≥ 0$, such that `value` $< 2^{8 * l}$, followed by an $8 * l$-bit unsigned big-endian representation of `value`. If `value` does not belong to the supported range, a range check exception is thrown. Returns that copy of the [`Builder{:tact}`][builder].
```

Comment:
> a range check exception is thrown

let's clarify which exception is exactly thrown and link to the exit codes page

Reply:
It's clarified right below that paragraph, but I'll remove the "range check exception" sentence in favor of that one below to separate description and possible errors